src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateTimeValue_TimeZone_Paris.txt

```diff
@@ -0,0 +1,11 @@
+﻿// (UTC+01:00) Brussels, Copenhagen, Madrid, Paris
+#SETUP: TimeZoneInfo(Romance Standard Time)
+
+>> DateTimeValue("jeudi 21 juillet 2022 19:34:03", "fr-FR")
+DateTime(2022,7,21,19,34,3,0)
+
+>> Text(DateAdd(DateTime(2022,3,13,1,0,0), 2, "hours"))
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

Can we add test comments explaining why these cases are significant?
Eg, "The result of this expression depends on current TimeZoneInfo. In Seattle TimeZoneInfo, this would trip over daylight savings and be  4:00am. In Paris timezoneinfo - which does not do daylight savings, we get 3:00am. "